csv-pricing.md

```diff
@@ -0,0 +1,118 @@
+# CSV Pricing
+
+> **Note**: This feature is only oficially supported on Kubecost Enterprise plans and during free trials.
+
+The following resource documents the steps required to set up custom prices with a CSV pipeline. This feature allows for individual assets (e.g. nodes) to be supplied at unique prices. Common uses are for on-premise machines or for external enterprise discounts. 
+
+1. Provide a file path for your CSV pricing data in values.yaml. This path can reference a local PV or an S3 bucket. 
+
+    ```
```

Comment:
YAML syntax is very important. We need to be sure to retain the proper indentation and spacing.
The block should be formatted as follows:

``` yaml
pricingCsv:
  enabled: true
  location:
    provider: "AWS|GCP"
    region: "us-east-1"
    URI: < your-valid-file-URI >
    csvAccessCredentials: pricing-schema-access-secret
```

Reply:
Fair enough, we had to transcribe a lot of this from a GDoc so some of the formatting is still a little funky. Good catch.

Reply:
Yeah... Pasting from GDocs is hit or miss. Especially when you strip formatting.

Reply:
I mean I'm not just dropping it in, it still gets fixed up but yeah spacing is the easiest thing that can get warped in the process